backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/TeamService.cs

```diff
@@ -124,19 +126,23 @@ public async Task<List<TeamDto>> GetCurrentUserTeams()
             .ToListAsync();
 
         return _mapper.Map<List<TeamDto>>(teams);
-    } 
+    }
     public async Task<ICollection<NewMeetingMemberDto>> GetTeamMembersOfCurrentUserAsync()
     {
         var currentUser = await _userService.GetCurrentUserAsync();
+
         var teamMembers = await _context.Users
-            .Include(x => x.TeamMembers)
             .Where(x => x.Id == currentUser.Id)
-            .Select(x => _mapper.Map<NewMeetingMemberDto>(x))
+            .Include(x => x.TeamMembers)
+                .ThenInclude(x => x.Team)
+                .ThenInclude(x => x.TeamMembers)
```

Comment:
Why do we include team members two times?

Reply:
Fixed it.